### PR TITLE
Cypress

### PR DESCRIPTION
**Pull request by carolinwagner**
Even though it is not a regular user story but just cypress tests and small styling changes, I would really appreciate a review. 

Closes #10

### CODE REVIEW

File: cypress/integration/QuestionForm.spec.js [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    cy.title().should('include', 'hello')
+  })
+
+  it('has a headline', () => {
+    cy.get('h1').should('have.length', 1)
+  })
+
+  it('has a button', () => {
+    cy.get('button').contains('Zusammenfassung anzeigen').click()
+  })
```
Comment (mpagels) [inline cypress/integration/QuestionForm.spec.js:18]: why do you click the button in the test?

File: cypress/integration/QuestionForm.spec.js [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+    cy.get('p').contains('absolute Mehrheit')
+
+    cy.get('li').contains('Tag soll über die Satzung beschlossen')
+    cy.get('p').contains('2020-12-31')
+
+    cy.get('li').contains('Mitglieder sollen die Satzung untersch')
+    cy.get('p').contains('10')
+  })
+
+  it.only('shows error messages if no input is set', () => {
```
Comment (mpagels) [inline cypress/integration/QuestionForm.spec.js:265]:
```suggestion
  it('shows error messages if no input is set', () => {
```

**Review by mpagels — COMMENTED** [review]
Have one question, pls see review.
And I have an idea for another test for your QuestionForm ;-)
You could test, if the form validates the inputs correct. Right now you test only a complete empty form. But you could test, if every validation for every input works. ;-)

File: cypress/integration/QuestionForm.spec.js (hunk 1 above)
Comment (carolinwagner) [reply cypress/integration/QuestionForm.spec.js:18]: Hmm good question - will remove it!

File: cypress/integration/QuestionForm.spec.js (hunk 2 above)
Comment (carolinwagner) [reply cypress/integration/QuestionForm.spec.js:265]: oops, thanks 😄

**Comment by carolinwagner** [comment]
Thanks for the review and the test suggestion @mpagels :-) I added two more test for number and date browser validation

**Pull request merged**